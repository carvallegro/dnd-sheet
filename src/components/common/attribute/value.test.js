import React from 'react'
import { shallow } from 'enzyme'

import Value from './value'

describe('<Value />', () => {
  it('renders', () => {
    const wrapper = shallow(<Value value={1} />)
    expect(wrapper).toMatchSnapshot()
  })

  it('renders with negative value', () => {
    const wrapper = shallow(<Value value={-2} />)
    expect(wrapper).toMatchSnapshot()
  })
})
