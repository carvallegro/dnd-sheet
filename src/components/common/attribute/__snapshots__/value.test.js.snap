// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Value /> renders 1`] = `
<styled.div>
  <styled.p>
    1
  </styled.p>
  <styled.img
    src="sub_shield.svg"
  />
</styled.div>
`;

exports[`<Value /> renders with negative value 1`] = `
<styled.div>
  <styled.p>
    -2
  </styled.p>
  <styled.img
    src="sub_shield.svg"
  />
</styled.div>
`;
